  challenge_loader_python.cpp
c:\proyectos\proyectos09\secureworld\challenges_py\challenge_loader_python-nokia\challenge_loader_python\challenge_loader_python.cpp(66): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
c:\proyectos\proyectos09\secureworld\challenges_py\challenge_loader_python-nokia\challenge_loader_python\challenge_loader_python.cpp(90): warning C4018: '<': signed/unsigned mismatch
c:\proyectos\proyectos09\secureworld\challenges_py\challenge_loader_python-nokia\challenge_loader_python\challenge_loader_python.cpp(99): warning C4477: 'printf' : format string '%d' requires an argument of type 'int', but variadic argument 1 has type 'size_t'
  c:\proyectos\proyectos09\secureworld\challenges_py\challenge_loader_python-nokia\challenge_loader_python\challenge_loader_python.cpp(99): note: consider using '%zd' in the format string
  dllmain.cpp
  pch.cpp
     Creating library C:\proyectos\proyectos09\SECUREWORLD\challenges_py\CHALLENGE_LOADER_PYTHON-NOKIA\challenge_loader_python\x64\Release\challenge_loader_python.lib and object C:\proyectos\proyectos09\SECUREWORLD\challenges_py\CHALLENGE_LOADER_PYTHON-NOKIA\challenge_loader_python\x64\Release\challenge_loader_python.exp
  Generating code
  All 12 functions were compiled because no usable IPDB/IOBJ from previous compilation was found.
  Finished generating code
  challenge_loader_python.vcxproj -> C:\proyectos\proyectos09\SECUREWORLD\challenges_py\CHALLENGE_LOADER_PYTHON-NOKIA\challenge_loader_python\x64\Release\challenge_loader_python.dll
